sklearn/metrics/classification.py

```diff
@@ -184,6 +184,89 @@ def accuracy_score(y_true, y_pred, normalize=True, sample_weight=None):
     return _weighted_sum(score, sample_weight, normalize)
 
 
+def top_k_accuracy_score(y_true, y_pred, k=5, normalize=True):
+    """Top k Accuracy classification score.
+    For multiclass classification tasks, this metric returns the
+    number of times that the correct class was among the top k classes
+    predicted.
+
+    Parameters
+    ----------
+    y_true : 1d array-like, or class indicator array / sparse matrix
```

Comment:
you should add corresponding tests (sparse and multidim y_true), I'm not sure the current implementation works on these cases